src/Control/Monad/Linear.hs

```diff
@@ -1,6 +1,27 @@
 {-# LANGUAGE LinearTypes #-}
 {-# LANGUAGE NoImplicitPrelude #-}
 
+
+-- | This module defines the linear monad hierarchy and classic monad instances.
+--
+-- It's critical to know that the linear monad hierarchy is different from
+-- the "Data.Control.Functor" hierarchy.
+-- To understand the differences see this [blog post](https://www.tweag.io/posts/2020-01-16-data-vs-control.html).
+--
+-- You should use
+--
+-- * control functors for an effectful computation holding a single @a@ in a @f a@.
+-- * control applicative functors when you want a control functor that you can apply
+-- * linear monads when you want to sequence /linear/ functions with a context, i.e.,
+--   when you want to sequence functions of the form @a #-> m b@.
+--
```

Comment:
I'm not sure these sentences carry any information unfortunately. Skip this for now. It's alright, linear-base is for fairly advanced users as it stands.
```suggestion
```